go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/k8snetworkplumbingwg/sriov-network-operator
 
-go 1.22
+go 1.22.4
```

Comment:
@bn222 why do we need to require .4 explicitly ?

Reply:
gopls doesn't work with non-explicit z-stream. Also, our images downstream are 1.22.4. Making it explicit brings our env and upstream in alignment.

Side note: A difference between 1.22.1 and 1.22.4 causes code gen to behave differently. Making it explicit makes sure everyone is generating the same code.

Reply:
I don't like this approach. It will force us to. make more updates (1.22.5 is already releases). Could you please provide any real example how this will help?

Reply:
* gopls lsp servers doesn't work when you don't specify 1.22.4. There is an issue between 1.22.0 and 122.4. 
* I'm not recommending always bumping versions to z-stream. If 1.22.5, we don't necessarily want it.

Reply:
Also, As far as I know. version specified in the go.mod file is the _minum required version_ for this project to build.

https://go.dev/doc/modules/gomod-ref

So, there will be no need to bump this in the future in my opinion.

@bn222 do you have any reference (github issue, stackoverflow) to the problem with gopls?

Reply:
https://github.com/NixOS/nixpkgs/issues/293180

Reply:
That's the closest I can find.